# Implements SBP MSG_BASELINE_HEADING

uint32 tow    # GPS Time of Week
uint32 mdeg   # Heading
uint8 n_sats  # Number of satellites used in solution
uint8 flags   # Status ï¬‚ags (See SBP manual for details)
